### PR TITLE
update for 3D object render, revert old changes

### PR DESCRIPTION
**Pull request by liamcindy**
@xiaofeng11 @super626 This PR revert some unused changes, Please review this PR, thanks.

### CODE REVIEW

File: extensions/Particle3D/CCParticle3DRender.cpp [hunk 1]
```diff
@@ -213,6 +215,15 @@ bool Particle3DQuadRender::initQuadRender( const std::string& texFile )
     return true;
 }
 
+void Particle3DQuadRender::reset()
+{
+    auto tex = Director::getInstance()->getTextureCache()->getTextureForKey(_texFile);
+    if (tex != nullptr)
```
Comment (super626) [inline extensions/Particle3D/CCParticle3DRender.cpp]: Why if? Does this->initQuadRender(_texFile) work with wrong file name?

File: extensions/Particle3D/PU/CCPURender.cpp [hunk 2]
```diff
@@ -587,33 +595,18 @@ bool PUParticle3DEntityRender::initRender( const std::string &texFile )
...
-    return _texture->getName();
+    PURender::copyAttributesTo(render);
 }
 
-void PUParticle3DEntityRender::copyAttributesTo(PUParticle3DEntityRender *render)
+void PUParticle3DEntityRender::reset()
 {
-    PURender::copyAttributesTo(render);
+    auto tex = Director::getInstance()->getTextureCache()->getTextureForKey(_texFile);
+    if (tex != nullptr)
```
Comment (super626) [inline extensions/Particle3D/PU/CCPURender.cpp]: Can we remove if?

File: extensions/Particle3D/CCParticle3DRender.cpp (hunk 1 above)
Comment (liamcindy) [inline extensions/Particle3D/CCParticle3DRender.cpp]: It has diffrent render state with diffrent texture

**Comment by super626** [comment]
Ok for me. @xiaofeng11, @minggo , @pandamicro any suggestion?

**Comment by super626** [comment]
js can not compile. Please fix it.

**Comment by xiaofeng11** [comment]
It's OK for me

**Pull request merged**